test_runner/src/main/kotlin/ftl/run/exception/ExceptionHandler.kt

```diff
@@ -3,18 +3,23 @@ package ftl.run.exception
 import flank.common.logLn
 import ftl.json.SavedMatrix
 import ftl.run.cancelMatrices
-import ftl.util.captureError
+import ftl.util.report
 import kotlinx.coroutines.runBlocking
 import kotlin.system.exitProcess
 
 fun withGlobalExceptionHandling(block: () -> Int) {
+    withGlobalExceptionHandling(block, { exitProcess(it) })
+}
+
+// Overloading this function make tests easier to perform and tests exit code properly
```

Comment:
Usually we dont have comments like this.
Regardless the wording is strange.
```suggestion
// Overloading this function makes tests easier to implement and exit with the correct exit code
```